drop_flag_tool.py

```diff
@@ -0,0 +1,13 @@
+def calculate_drop_flag(flag):
```

Comment:
This file should be moved into Notes

Reply:
All of these will be removed/moved when I'm ready to mark this PR ready. For now I'll keep them for if/when I need to do testing. Will leave this comment open as a reminder.

Reply:
This file has been deleted. A separate file detailing how flags are stored was added to notes.